subscriptions-api.md

```diff
@@ -280,7 +281,7 @@ Each subscription is represented by an object that has the following properties:
   using the selected protocol. The format of the address MUST be valid for the
   selected protocol or one of the protocol's own transport bindings (e.g. AMQP
   over WebSockets).
-- **filter** - OPTIONAL. A filter is an expression of a particular filter
+- **filters** - OPTIONAL. One or more filter expressions of a particular filter
```

Comment:
Keep it as one

Reply:
It's one filter, but the filter may contain multiple conditions, as described [here](https://github.com/cloudevents/spec/blob/master/subscriptions-api-openapi.yaml). I think the confusion results from an inconsistency in [3.2.3.1](https://github.com/cloudevents/spec/blob/master/subscriptions-api.md#3231-basic-filter-dialect), where `filters` instead of `conditions` is used.